components/servo/fake-ld.sh

```diff
@@ -1,3 +1,15 @@
-#!/bin/bash
-TARGET_DIR=$OUT_DIR/../../..
-arm-linux-androideabi-gcc $@ $LDFLAGS -lc -o $TARGET_DIR/libservo.so -shared && touch $TARGET_DIR/servo
+#!/usr/bin/env bash
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+TARGET_DIR="${OUT_DIR}/../../.."
+arm-linux-androideabi-gcc "$@" \
+                          "${LDFLAGS}" -lc -shared \
```

Comment:
`LDFLAGS` is not always set, so we need to give it a default value if it isn't. A bash [parameter expansion](http://wiki.bash-hackers.org/syntax/pe#use_a_default_value) should do the trick.